src/test/unit_tests/jest_tests/test_nc_lifecycle_cli.test.js

```diff
@@ -809,8 +811,9 @@ describe('noobaa cli - lifecycle notifications', () => {
         await create_object(object_sdk, test_bucket, test_key1, 100, false);
         await create_object(object_sdk, test_bucket, test_key2, 100, false);
 
-        await exec_manage_cli(TYPES.LIFECYCLE, '', {disable_service_validation: 'true', disable_runtime_validation: 'true', config_root}, undefined,
-            {NOTIFICATION_LOG_DIR: tmp_lifecycle_logs_dir_path});
+        await config_fs.create_config_json_file(JSON.stringify({ NOTIFICATION_LOG_DIR: tmp_lifecycle_logs_dir_path }));
```

Comment:
if its the same for all notifications logs, maybe its better to move the beforeEach?